app/models/miq_region.rb

```diff
@@ -1,4 +1,6 @@
 class MiqRegion < ApplicationRecord
+  belongs_to :maintenance_zone, :class_name => 'Zone', :inverse_of => :maintenance_zone_region
```

Comment:
Do you need the `inverse_of`?

Reply:
rubocop needs inverse_of

Reply:
`inverse_of => false` will make rubocop happy